src/index.html

```diff
@@ -6,9 +6,81 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+
+      <nav class="header__menu">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a class="header__list-item-link is-active"
+              href="#apple">
```

Comment:
```suggestion
            <a
              class="header__list-item-link is-active"
              href="#apple"
            >
```

Reply:
<img width="1017" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/2438091c-9de8-497c-b76b-ef27add1c089">

Reply:
The same in all code